GitBasic/Controls/ConsoleControl.xaml

```diff
@@ -8,7 +8,7 @@
     d:DesignHeight="400"
     d:DesignWidth="800"
     mc:Ignorable="d">
-    <Grid>
+    <Grid DataContext="{Binding RelativeSource={RelativeSource AncestorType={x:Type local:ConsoleControl}}}">
```

Comment:
Similar to what I was trying to do in the meeting, but couldn't remember. This sets the data context of the root grid of  console control to the console control itself. This way I can data bind to properties in the code behind.